packages/database/lib/schema.ts

```diff
@@ -275,3 +297,31 @@ export const candles = pgTable('candles', {
 }, table => ({
   pk: primaryKey(table.marketAcct, table.candleDuration, table.timestamp)
 }));
+
+export const comments = pgTable('comments', {
+  // Need this as we reference this for response and nesting
+  commentId: bigint('comment_id', {mode: 'bigint'}).notNull(),
+  // Generated when comment is created
+  commentorAcct: pubkey('commentor_acct').notNull(),
+  proposalAcct: pubkey('proposal_acct').notNull(),
+  // This will be the body content of the comment
+  content: text('content').notNull(),
+  // Use only if its a responding comment in a chain, we should constrain this so
+  // it references only commentIds which have respondingCommentId with NULL
+  respondingCommentId: bigint('responding_comment_id', {mode: 'bigint'}).references(() => comments.commentId),
+  createdAt: timestamp('created_at').notNull().default(sql`now()`)
+}, table => ({
+  pk: primaryKey(table.commentId, table.commentorAcct, table.proposalAcct),
+  first: unique('commentor_content').on(table.commentorAcct, table.content)
```

Comment:
Should prevent spam.

Reply:
So I can't write `LGTM` on every proposal?

Reply:
Perhaps we should enforce anti spam at the API write layer instead of here

Reply:
Yea I just often exist in db land where if I can design at the data layer I do my best.

In general what I wanted to do was prevent exactly that, so a user cannot take an ad for "crypto bonus pool winning max 2002" and just copy paste. 

And to be fair a LGTM, use an emoji.....